tools/base/src/Base.ts

```diff
@@ -126,6 +133,28 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
             }
         }
 
+        protected adoptComponentFragments(): void {
+            const componentFragments = Theme.getComponentFragments(
+                this.constructor.name
```

Comment:
This will not work with bundlers that mangle names.

Reply:
Good catch, what if we use `this.tagName`?

Reply:
I went ahead and switched it over to `tagName`, let me know your thoughts.